components/database/couchdb-21/couchdb.p5m

```diff
@@ -0,0 +1,1336 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
+#
+
+<transform link path=usr/bin/(.+)     -> default pkg.linted true >
+<transform link path=usr/lib/$(MACH64)/couchdb/(.+) -> default pkg.linted true >
+<transform file path=usr/lib/$(MACH64)/couchdb/lib/couch-$(COMPONENT_VERSION)/priv/couch(.+) -> set mode 0555 >
+<transform file path=usr/lib/$(MACH64)/couchdb/lib/couch-$(COMPONENT_VERSION)/priv/spawnkillable/couch(.+) -> set mode 0555 >
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+# CouchDB 2.1.0 must have an erlang runtime < 19
+depend type=require fmri=runtime/erlang@17.5-2017.0.0.0
+depend type=require fmri=runtime/python-27
+depend type=require fmri=library/python/netaddr-27
+depend type=require fmri=database/couchdb-common
+
+dir  path=etc/couchdb/local.d owner=couchdb group=couchdb mode=0755
+
+file path=etc/couchdb/default.d/README
```

Comment:
Sorry for raising this late, but can couchdb follow the common layout of other DBMS on OI (MongoDB, PostgreSQL, MariaDB, etc)?
I mean: 
- we deliver binary files to /usr/couchdb/${MAJOR_VERSION},  so that we have /usr/couchdb/2.1/{bin,lib,etc...}
- we deliver data files directories in /var/couchdb/${MAJOR_VERSION}, 
- we create mediated links from /usr/bin, /usr/share/man to delivered files.

Reply:
No problem.
In fact I have couchdb-16 also so we can test two versions.
I have no experience with mediators but we will see.
But I will need some time.